client/src/chat/chatApiClient.ts

```diff
@@ -35,26 +35,28 @@ class ChatApiClient extends ApiClientBase {
 		// I'm confused about how to use the WsMessageEvent
```

Comment:
This was basically intended as a way for the server to communicate to the client in a consistent format, but it works for the client too to server as well. For right now, we just have one client->server ws message. What if we had multiple? Do we open up a connection for each?  I think there's some overhead there, so I was trying to standardize how such communication occurred, so each message was guaranteed to have certain properties regardless of the operation (e.g., 'dataType' field says what kind of thing we are modifying)

Reply:
Yeah, i got this in the end